repos/system_upgrade/el7toel8/actors/nischeck/libraries/nischeck.py

```diff
@@ -0,0 +1,152 @@
+import re
+import ipaddress
```

Comment:
This violates import order, please rebase your patch and run `make lint_fix` to fix isort violations.